apps/remote_control/lib/mix/tasks/namespace/path.ex

```diff
@@ -24,7 +24,11 @@ defmodule Mix.Tasks.Namespace.Path do
         |> Namespace.Module.apply()
         |> Atom.to_string()
 
-      String.replace(path, string_name, namespaced_name)
+      if String.contains?(path, namespaced_name) do
```

Comment:
this prevents double namespacing. I saw that we were generating `lx_lx_protocol.app`